src/main/java/org/jabref/gui/exporter/ExportCommand.java

```diff
@@ -114,14 +119,32 @@ private void export(Path file, FileChooser.ExtensionFilter selectedExtensionFilt
         preferences.getImportExportPreferences().setExportWorkingDirectory(file.getParent());
 
         final List<BibEntry> finEntries = entries;
+
+        DialogPane dialogPane = new DialogPane();
+        Optional<ButtonType> buttonType = dialogService.showCustomDialogAndWait(
```

Comment:
Please use  Localization correectly https://devdocs.jabref.org/getting-into-the-code/code-howtos#using-localization-correctly